packages/govuk-frontend/src/govuk/common/normalise-string.mjs

```diff
@@ -24,19 +25,36 @@ export function normaliseString(value) {
 
   const trimmedValue = value.trim()
 
-  if (trimmedValue === 'true') {
-    return true
-  }
+  // No schema type set? Determine automatically
```

Comment:
Nah it's not happening at all and I LOVE this comment

Because neither are the checks for objects, arrays and non-finite numbers ever happening 😣

Can we make `normaliseString()` only deal with strings from `DOMStringMap`? (As the name suggests)

Only strings ever get passed in so we can [remove the confusing ternary](https://github.com/alphagov/govuk-frontend/pull/4792#discussion_r1517528279)

Reply:
I think it'd be well fine for `normaliseString` to only deal with string inputs (whether from `DOMStringMap` or not), as that's the only thing we use it for.

Reply:
Especially if it lets us remove unwieldly ternaries 😁

Reply:
Ah for nested objects like `config.i18n` we have no schema, so the types are still automatic:

https://github.com/alphagov/govuk-frontend/blob/4a40d639dc8eca1e301a7b624cdbb3ce44347b4d/packages/govuk-frontend/src/govuk/common/index.mjs#L96-L98

Reply:
Oh, indeed, good spot.